bigquery_etl/view/__init__.py

```diff
@@ -190,14 +190,13 @@ def _valid_view_naming(self):
             for t in parsed.tokens
             if not (t.is_whitespace or isinstance(t, sqlparse.sql.Comment))
         ]
-        is_view_statement = (
-            " ".join(tokens[0].normalized.split()) == "CREATE OR REPLACE"
-            and (
-                tokens[1].normalized == "VIEW"
-                or (
-                    tokens[1].normalized.upper() == "MATERIALIZED"
-                    and tokens[2].normalized == "VIEW"
-                )
+        is_view_statement = " ".join(
+            tokens[0].normalized.split()
+        ) == "CREATE OR REPLACE" and (
+            tokens[1].normalized == "VIEW"
+            or (
+                tokens[1].normalized.upper() == "MATERIALIZED"
+                and tokens[2].normalized == "VIEW"
             )
         )
```

Comment:
Generally I'm fine with the formatting Black requires, but then occasionally it does something like this, which I find so much less readable. 😞